lib/OpenQA/CacheService/Task/Asset.pm

```diff
@@ -33,13 +33,14 @@ sub _cache_asset {
     return $job->finish unless defined $asset_name && defined $type && defined $host && defined $lock;
     my $guard = $app->progress->guard($lock);
     unless ($guard) {
-        $job->note(output => 'Asset was already requested by another job');
+        $job->note(
+            output => "Asset $asset_name was downloaded by another job, details are therefore unavailable to us");
         return $job->finish;
     }
 
     my $log = $app->log;
     my $ctx = $log->context('[#' . $job->id . ']');
-    $ctx->info("Download: $asset_name");
+    $ctx->info("Downloading $asset_name");
```

Comment:
Here I would also prefer the colon or additional quotes.

Reply:
I totally don't care, can we get a vote on this? 😆